### PR TITLE
Sfl 86

### PR DESCRIPTION
**Pull request by saketPandhare-Webonise**
At some places one line blank space is purposefully kept just to avoid confusion while reading

### CODE REVIEW

File: SUNFLOWR/SUNFLOWR/FilterVC/Filter+TableView.swift [hunk 1]
```diff
@@ -87,14 +87,78 @@ extension FilterVC: UITableViewDelegate, UITableViewDataSource, TableCellDelegat
...
+    /// This function is used to configure filters for the API request
+    /// - Parameter indexPath: indexPath
+    func configureFiltersForAPIRequest(indexPath: IndexPath, cell: FilterTableCell, shouldAdd: Bool) {
+        if (indexPath.section == 1) {
+            ///time_range
+            ///Perform Time Filter Action
+            updateTimeFilter(shouldAdd: shouldAdd, cell: cell)
+        }
+        else if (indexPath.section == 2){
+            ///done": true,"not_done": true,"favorite": true
```
Comment (vineet-webonise) [inline SUNFLOWR/SUNFLOWR/FilterVC/Filter+TableView.swift:111]: remove commented code if not needed

File: SUNFLOWR/SUNFLOWR/FilterVC/Filter+TableView.swift [hunk 2]
```diff
@@ -87,14 +87,78 @@ extension FilterVC: UITableViewDelegate, UITableViewDataSource, TableCellDelegat
...
+        let timeText = formatTimeFilterAsRequiredForAPI(time: cell.lblFilterName.text ?? "")
+        if (shouldAdd) {
+            arraySelectedTimeFilter.append(timeText)
+        }else {
+            arraySelectedTimeFilter = arraySelectedTimeFilter.filter {$0 != timeText}
+        }
+    }
+    
+    func updateClassStatus(rowNo: Int, cell: FilterTableCell) {
+        if (rowNo == 0){ ///this is completed check box
```
Comment (vineet-webonise) [inline SUNFLOWR/SUNFLOWR/FilterVC/Filter+TableView.swift:147]: Use switch statement here.

File: SUNFLOWR/SUNFLOWR/MindVC/MindVC.swift [hunk 3]
```diff
@@ -285,6 +285,16 @@ extension MindVC: DidReceiveMindDelegate {
 }
 
 extension MindVC: FilterScreenDelegate {
+    
+    func toggleFilterBtn(arraySelectedFilters: [IndexPath], arrayFilterValues: [String], searchString: String, arrayTimeRange: [String], done: Bool, notDne: Bool, fav: Bool) {
+        ///ToDo: while toggling on Filter screen
+    }
+    
+
```
Comment (vineet-webonise) [inline SUNFLOWR/SUNFLOWR/MindVC/MindVC.swift:293]: remove blank space

File: SUNFLOWR/SUNFLOWR/MindVC/Mind+Filter.swift [hunk 4]
```diff
@@ -0,0 +1,10 @@
+//
+//  Mind+Filter.swift
+//  SUNFLOWR
+
+import Foundation
+import UIKit
+
+extension MindVC {
+
```
Comment (vineet-webonise) [inline SUNFLOWR/SUNFLOWR/MindVC/Mind+Filter.swift:9]: Will this extension be used in future? As its not having any body.

File: SUNFLOWR/SUNFLOWR/MoveVC/MoveVC.swift [hunk 5]
```diff
@@ -41,16 +49,16 @@ class MoveVC: NavigationController {
         configureTableCells()
         editActionVM.getCurrentWeekProgram(view: self.view)
         configureFilterIcon()
+       // moveVM.callAPIForFiltering()
```
Comment (vineet-webonise) [inline SUNFLOWR/SUNFLOWR/MoveVC/MoveVC.swift:52]: remove commented code if not required

Comment (saketPandhare-Webonise) [reply SUNFLOWR/SUNFLOWR/MoveVC/MoveVC.swift:52]: Done

File: SUNFLOWR/SUNFLOWR/MindVC/MindVC.swift (hunk 3 above)
Comment (saketPandhare-Webonise) [reply SUNFLOWR/SUNFLOWR/MindVC/MindVC.swift:293]: Done

File: SUNFLOWR/SUNFLOWR/MindVC/Mind+Filter.swift (hunk 4 above)
Comment (saketPandhare-Webonise) [reply SUNFLOWR/SUNFLOWR/MindVC/Mind+Filter.swift:9]: Yes

File: SUNFLOWR/SUNFLOWR/FilterVC/Filter+TableView.swift (hunk 2 above)
Comment (saketPandhare-Webonise) [reply SUNFLOWR/SUNFLOWR/FilterVC/Filter+TableView.swift:147]: return in "if" statement will act in same was as switch ,only one will get execute at time

File: SUNFLOWR/SUNFLOWR/FilterVC/Filter+TableView.swift (hunk 1 above)
Comment (saketPandhare-Webonise) [reply SUNFLOWR/SUNFLOWR/FilterVC/Filter+TableView.swift:111]: This purposefully kept for reference to avoid confusion when we revisit it

**Pull request merged**